avocado/core/nrunner.py

```diff
@@ -928,10 +934,9 @@ def are_requirements_available(self, runners_registry=None):
             runners_registry = RUNNERS_REGISTRY_STANDALONE_EXECUTABLE
         return self.runnable.pick_runner_command(runners_registry)
 
-    def setup_output_dir(self):
-        self.output_dir = tempfile.mkdtemp(prefix='.avocado-task-')
-        env_var = {'AVOCADO_TEST_OUTPUTDIR': self.output_dir}
```

Comment:
This breaks https://avocado-framework.readthedocs.io/en/latest/guides/writer/chapters/writing.html#environment-variables-for-tests for exec-tests.

Reply:
Thanks, I will investigate it.